apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -187,16 +171,24 @@ export const CourseBlocksAll = React.createClass({
         {!this.props.isEnglish && (
           <ProtectedStatefulDiv ref="csfCoursesHeader"/>
         )}
-        <CourseBlocksCsf isEnglish={this.props.isEnglish}/>
 
```

Comment:
I would say this is somewhat personal preference, and either approach is probably okay (I personally prefer to do as you suggest and destructure props at the beginning of the render).